third_party/terraform/tests/resource_logging_billing_account_exclusion_test.go

```diff
@@ -165,3 +143,18 @@ resource "google_logging_billing_account_exclusion" "basic" {
 }
 `, exclusionName, billingAccount, description, getTestProjectFromEnv())
 }
+
+func testAccLoggingBillingAccountExclusion_multipleCfg(billingAccount string) string {
+	s := ""
+	for i := 0; i < 3; i++ {
+		s += fmt.Sprintf(`
+resource "google_logging_billing_account_exclusion" "basic%d" {
```

Comment:
nice

Reply:
Oh right that just reminded me I was going to do the same for the project test, just pushed it